src/schema/v2/partner/partner.ts

```diff
@@ -455,14 +455,19 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
 
           const { body, headers } = await partnerArtworksLoader(id, gravityArgs)
           const totalCount = parseInt(headers["x-total-count"] || "0", 10)
+          const artworkIds = body.map((artwork) => artwork._id)
 
           // Only accept shallow = false argument if requesting user is authorized admin/partner
-          if (args.shallow === false && partnerArtworksAllLoader) {
-            const artworkIds = body.map((artwork) => artwork._id)
+          if (
+            args.shallow === false &&
+            artworkIds.length > 0 &&
```

Comment:
This solves the issue for this particular case, but I still think [MP should not drop query params that are empty arrays](https://artsy.slack.com/archives/CP9P4KR35/p1588364384042800). Gravity handles the empty `artwork_id` array param correctly. It'll be nice to try out the [new qs `allowEmptyArrays` option](https://github.com/ljharb/qs/pull/487) and fix this globally. The package hasn't been releasing new versions for 8 months so we might need to pull from github directly.

Reply:
Followup ticket [here](https://artsyproduct.atlassian.net/browse/EMI-1658).